SwiftLint.podspec

```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   s.preserve_paths            = '*'
   s.exclude_files             = '**/file.zip'
   s.ios.deployment_target     = '9.0'
-  s.macos.deployment_target   = '10.10'
+  s.macos.deployment_target   = '10.11'
```

Comment:
This should probably be done in a separate PR.
By the way, package.swift targets 10.12, so shouldn't we use 10.12 in podspec as well ?
@jpsim WDYT?

Reply:
This is a bit confusing, because the SwiftLint podspec just vends the SwiftLint library. SwiftLint doesn't run on iOS, watchOS or tvOS either, yet this podspec claims to support those platforms.

That's done so iOS/nacOS/tvOS/watchOS Xcode projects using CocoaPods can all add SwiftLint as a dependency, not to embed it into their apps but to run as an Xcode build phase.

So the deployment target here doesn't have to match what's in Package.swift and in fact should just go as far back as possible.

Reply:
I see. Thanks for detailed explanation. 
Change to 10.11 stems from using [Down](https://github.com/johnxnguyen/Down). It's far enough, isn't it ?

Reply:
Yes.